test/test_forest_plot.py

```diff
@@ -198,3 +183,116 @@ def generic_args(self, plot_var="plot_variable"):
                 unit_dict_display,
                 app_path,
                 init_time)
+
```

Comment:
Lets chat through these tests. They feel kind of fragile but buy the time I've got this far I'm feeling tired so not sure I'm clear on what's going on and what we might want to do instead.